commands/commands.go

```diff
@@ -115,22 +192,71 @@ func exists(path string) (bool, error) {
 	return false, err
 }
 
+func findFile(root string, fileName string) (string, error) {
```

Comment:
as much as you may find the old findFile "unreadable", that's actually the correct way to write go
"if-elsing" to death until you exhausted all the cases